core/app/models/spree/shipping_method_stock_location.rb

```diff
@@ -0,0 +1,6 @@
+class ShippingMethodStockLocation
+  self.table_name = :spree_shipping_methods_stock_locations
```

Comment:
:-1: here. This should be `Spree` namespaced (*eyeroll* for not writing `Solidus`) and the table name should be `spree_shipping_method_stock_locations`.